obc/drivers/vn100/vn100_common.h

```diff
@@ -0,0 +1,62 @@
+#pragma once
+
+#include "obc_errors.h"
+#include "vn100_binary.h"
+
+#include <stdint.h>
+
+/* To access the user manual for VN-100 click here --> https://geo-matching.com/media/migrationpiwnum.pdf */
+
+/**
+ * @brief Initiates VN100 with the default parameters for baudrate, polling rate, etc
+ * Defaults:
+ * User Tag: NULL
+ * Serial Baud Rate: 115200
+ * Async Data Output Frequency: 10 Hz
+ * Async Data Output Type: INS_LLA
+ * Syncronization control: 3,0,0,0,6,1,0,100000000,0
+ *
+ */
+void initVn100(void);
```

Comment:
extreme nit since other functions are prefixed with vn100:
```suggestion
void vn100Init(void);
```

Reply:
I personally pref the existing one because it follows the general naming convention of all other init functions in the repo atm. Think it would be strange to make an exception for this one?

Reply:
fair. ya i don't really mind that much either way